server/data/educationAndWorkPlanClient.ts

```diff
@@ -114,4 +117,17 @@ export default class EducationAndWorkPlanClient {
       data: createInductionRequest,
     })
   }
+
+  async getEducationResponse(prisonNumber: string, token: string): Promise<EducationResponse> {
+    return EducationAndWorkPlanClient.restClient(token).get<EducationResponse>({
+      path: `/person/${prisonNumber}/education`,
+    })
+  }
+
+  // Use this to test the view for education outside an induction
```

Comment:
I think there are better/different ways of doing this; but either way can we get rid of these comments/code, as we dont really want this kicking around in our main production code?
(same comment for all the similar commented out code; plus the `mockEducationData` file (that's what the test data builder file is for 😁 )

Reply:
Yes this was a bad idea! The intention was in case it needed to be demoed locally first, then delete before merging.